AdventOfCode2022/Program.cs

```diff
@@ -1,5 +1,60 @@
-﻿class Program
+﻿using System.Drawing;
+
+class Program
 {
+    List<List<int>> DistinctList(List<List<int>> coorSave)
+    {
+        List<List<int>> result = new List<List<int>>();
+        foreach (var coor in coorSave)
+        {
+            if (result.Count == 0)
+                result.Add(coor);
+            else
+            {
+                bool check = false;
+                foreach (var item in result)
+                {
+                    if ((item[0] == coor[0]) && (item[1] == coor[1]))
+                        check = true;
+                }
+                if (check == false)
+                    result.Add(coor);
+            }
+        }
+        return result;
+    }
+    
+    List<int> Check(List<int> coorH, List<int> coorT)
+    {
+        if ((Math.Abs(coorH[0] - coorT[0]) >= 1) && (Math.Abs(coorH[1] - coorT[1]) > 1) ||
+            (Math.Abs(coorH[0] - coorT[0]) > 1) && (Math.Abs(coorH[1] - coorT[1]) >= 1))
+        {
+            if (coorH[0] - coorT[0] > 0)
+                coorT[0]++;
+            else
+                coorT[0]--;
```

Comment:
you could use ternary conditional operator for such ifs, e.g.
```suggestion
            coorT[0] = coorH[0] - coorT[0] > 0 ? ++coorT[0] : --coorT[0];
```